pages/index.tsx

```diff
@@ -1,31 +1,39 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
 import Image from 'next/image';
+import Link from 'next/link';
 
 const Home: NextPage = () => {
   return (
-    <div className='flex min-h-screen flex-col items-center justify-center py-2'>
+    <div className='min-h-screenpy-2'>
       <Head>
-        <title>Defaang</title>
+        <title>defaang</title>
       </Head>
-
-      <main className='w-full  px-20 text-center'>
-        <h1 className='text-5xl font-medium'>
-          Hello There! Welcome to defaang. <br />A website that will curate recently-asked interview questions from
-          FAANG+ to help people practice & prep!
-        </h1>
-        <br />
-        <p className='font-medium text-teal-500'>
+      <div className='flex items-center justify-between py-4 shadow-[rgba(17,17,26,0.1)_0px_1px_0px] lg:pl-[20%] lg:pr-[20%]'>
+        <div className='cursor-pointer px-5 text-3xl font-bold text-gray-700'>defaang</div>
+        <div className='mx-5 flex w-1/3 items-center justify-between lg:w-1/6'>
+          <Link href='https://github.com/ykdojo/defaang' rel='noreferrer'>
+            <Image src='/github.svg' className='cursor-pointer' height={30} width={30} alt='github' />
+          </Link>
+          <Link href='https://discord.gg/aJp6Fypb' rel='noreferrer'>
+            <Image src='/discord.svg' className='cursor-pointer' height={30} width={30} alt='discord' />
+          </Link>
+          <Link href='https://twitter.com/ykdojo' rel='noreferrer'>
+            <Image src='/twitter.svg' className='cursor-pointer' height={30} width={30} alt='twitter' />
+          </Link>
+        </div>
+      </div>
+      <main className='flex h-[calc(100vh-4rem)] w-full flex-col items-center justify-center px-[5%]  text-center lg:h-[calc(100vh-6rem)] lg:px-[20%]'>
+        <h3 className='text-3xl font-bold text-gray-800 lg:text-5xl'>Welcome to defaang 🎉</h3>
```

Comment:
An `h1` would be more semantic here :smiley:

Reply:
ok